### PR TITLE
Adoption of some packages from worldofpeace

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122712` at 74e11d12 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>celluloid</li>
    <li>notejot</li>
    <li>taxi</li>
  </ul>
</details>

File: pkgs/applications/misc/notejot/default.nix [hunk 1]
```diff
@@ -20,31 +35,30 @@ stdenv.mkDerivation rec {
...
+    libadwaita
     libgee
     pantheon.elementary-icon-theme
     pantheon.granite
   ];
 
   postPatch = ''
-    patchShebangs meson/post_install.py
+    patchShebangs build-aux/post_install.py
+    chmod +x build-aux/post_install.py
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/notejot/default.nix:50]:
```suggestion
    chmod +x build-aux/post_install.py
    patchShebangs build-aux/post_install.py
```

File: pkgs/applications/video/celluloid/default.nix [hunk 2]
```diff
@@ -50,22 +50,20 @@ stdenv.mkDerivation rec {
...
-    homepage = "https://github.com/celluloid-player/celluloid";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ ];
+    maintainers = with maintainers; [ AndersonTorres ];
     platforms = platforms.linux;
   };
+
+  passthru.updateScript = nix-update-script {
+    attrPath = pname;
+  };
```
Comment (SuperSandro2000) [inline pkgs/applications/video/celluloid/default.nix:68]: Can we just use nix-update instead?

Comment (AndersonTorres) [reply pkgs/applications/video/celluloid/default.nix:68]: How?

Comment (SuperSandro2000) [reply pkgs/applications/video/celluloid/default.nix:68]: Just run ``nix-update celluloid`` in the root of the repo without any modifications.

Comment (AndersonTorres) [reply pkgs/applications/video/celluloid/default.nix:68]: In other words, get rid of these lines? Looks fine to me.

**Pull request merged**